app/components/views/Offset/ProjectSearch/ProjectSearchForm/index.tsx

```diff
@@ -54,7 +54,7 @@ export const ProjectSearchForm: FC<Props> = (props) => {
       (project) =>
         Number(
           project[`balance${selectedRetirementToken}` as BalanceAttribute]
-        ) > 0
+        ) > 1
```

Comment:
would it make more sense to add this in the subgraph query, so it returns less results?